src/style.css

```
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}

```

```suggestion
.stars {
}
```

redundant, rewrite all stars to span

---

Thanks, resolved